doc/ref_model/chapters/chapter03.md

```diff
@@ -539,6 +542,150 @@ SyncE was standardized by the ITU-T, in cooperation with IEEE, as three recommen
 * ITU-T Rec. G.8264 that describes the specification of Ethernet Synchronization Messaging Channel (ESMC)
 SyncE architecture minimally requires replacement of the internal clock of the Ethernet card by a phase locked loop in order to feed the Ethernet PHY. 
 
+
+<a name="3.5.7"></a>
+ ### 3.5.7 Kubernetes Networking Semantics
+The support for traditional network orchestration is more or less non existing in Kubernetes proper. Kubernetes is foremost a Platform as a Service (PaaS) environment and not an Infrastrutcure as a Service (Iaas) infrastucture component. There is no orchetration API like Neutron and no way to instantiate network services such as L3aaS and LBaaS like you can do in Openstack.
```

Comment:
The sentence "Kubernetes is foremost a Platform as a Service (PaaS) environment and not an Infrastrutcure as a Service (Iaas) infrastucture component." kind of hangs -- expecting something else to follow.

Also, typo's in " infrastucture) --> " infrastructure" (2  places)

Reply:
I think Kubernetes is a bit of IaaS and a bit of SaaS, but inperfect in both roles. I would not state clearly that it is SaaS and not IaaS. I agree that network orchestration is missing though.

Reply:
Openstack -> OpenStack